app/src/main/java/com/zeapo/pwdstore/git/sshj/OpenKeychainWrappedKeyAlgorithmFactory.kt

```diff
@@ -69,15 +69,15 @@ class OpenKeychainWrappedSignature(private val wrappedSignature: Signature, priv
         }
     }
 
-    override fun sign() = if (bridgedPrivateKey != null) {
+    override fun sign(): ByteArray = if (bridgedPrivateKey != null) {
```

Comment:
We may want to allow null here just in case the Signature objects we wrap may return those.